src/main/java/com/javacode/entities/Comment.java

```diff
@@ -0,0 +1,87 @@
+package com.javacode.entities;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "comments")
+public class Comment {
+	@Id
+	@Column(name = "id")
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int id;
+
+	@Column(name = "content")
+	private String content;
+
+	@Column(name = "reply_id")
+	private int reply_id;
+
+	@OneToOne(cascade = CascadeType.ALL)
```

Comment:
`cascade = CascadeType.ALL` xóa đi, có cần mô nhé

update bằng tay chứ ko để auto nhé gái

Reply:
Mà sao onetoone ta? phải là `many-to-one` chứ cô ơi

Reply:
Dạ vâng ạ